src/main/resources/resources/localization/english_en-us.json

```diff
@@ -4,21 +4,19 @@
   "minicraft.achievement.benchmarking": "Benchmarking",
   "minicraft.achievement.benchmarking.desc": "Make a workbench.",
   "minicraft.achievement.upgrade": "Upgrade!",
-  "minicraft.achievement.upgrade.desc": "Get a tool with a stronger material than wood.",
+  "minicraft.achievement.upgrade.desc": "Get any Stone tool.",
   "minicraft.achievement.bow": "Bow down to me!",
-  "minicraft.achievement.bow.desc": "Kill a Mob with a bow.",
+  "minicraft.achievement.bow.desc": "Fire an arrow with a bow.",
   "minicraft.achievement.fish": "Go Fish!",
   "minicraft.achievement.fish.desc": "Fish up a Fish!",
   "minicraft.achievement.doors": "Adooring Protection",
-  "minicraft.achievement.doors.desc": "Place all door types.",
+  "minicraft.achievement.doors.desc": "Craft a Wood Door.",
   "minicraft.achievement.planks": "Walk the Planks!",
-  "minicraft.achievement.planks.desc": "Place all plank types.",
+  "minicraft.achievement.planks.desc": "Craft Wood Planks.",
   "minicraft.achievement.clothes": "Have a colourful day!",
-  "minicraft.achievement.clothes.desc": "Craft all colored clothes.",
+  "minicraft.achievement.clothes.desc": "Craft any color of Clothes",
```

Comment:
Stone, Wood Door, Wood Planks, and Clothes should not be capitalized. They weren't before, and to be honest I don't know why you did that.
As a general rule of thumb, only titles, first word in a sentence, country names, and some other words like "I" should be capitalized. For most words, it shouldn't.
I know nouns are capitalized in several other languages, like German, so I understand the confusion.

Reply:
Yeah this is an issue I have. I tend to CAP things that stick out to me. Only happens when I’m typing, not writing.